packages/client/src/views/UserAudit/UserAudit.tsx

```diff
@@ -0,0 +1,329 @@
+
+                  <ToggleMenu
+                    id={`sub-page-header-munu-button`}
+                    toggleButton={<VerticalThreeDots />}
+                    menuItems={getMenuItems(
+                      user.id as string,
+                      user.status as string
+                    )}
+                    hide={(scope && !scope.includes('sysadmin')) || false}
+                  />
+                ]}
+                size={ContentSize.LARGE}
+              >
+                <ContentWrapper>
+                  <InformationHolder>
+                    <InformationTitle>
+                      {intl.formatMessage(userSetupMessages.assignedOffice)}
+                    </InformationTitle>
+                    <InformationValue>
+                      <LinkButtonWithoutSpacing
+                        id="office-link"
+                        onClick={() =>
+                          dispatch(goToTeamUserList(user.primaryOffice!.id))
+                        }
+                      >
+                        {user.primaryOffice && user.primaryOffice.displayLabel}
+                      </LinkButtonWithoutSpacing>
+                    </InformationValue>
+                  </InformationHolder>
+                  <InformationHolder>
+                    <InformationTitle>
+                      {(userType &&
+                        intl.formatMessage(userSetupMessages.roleType)) ||
+                        intl.formatMessage(userFormMessages.labelRole)}
+                    </InformationTitle>
+                    <InformationValue>
+                      {(userType && `${userRole} / ${userType}`) || userRole}
+                    </InformationValue>
+                  </InformationHolder>
+                  <InformationHolder>
+                    <InformationTitle>
+                      {intl.formatMessage(userSetupMessages.phoneNumber)}
+                    </InformationTitle>
+                    <InformationValue>{user.number}</InformationValue>
+                  </InformationHolder>
+                  <InformationHolder>
+                    <InformationTitle>
+                      {intl.formatMessage(userSetupMessages.nid)}
+                    </InformationTitle>
+                    <InformationValue>{user.nid}</InformationValue>
+                  </InformationHolder>
+                  <InformationHolder>
+                    <InformationTitle>
+                      {intl.formatMessage(userSetupMessages.userName)}
+                    </InformationTitle>
+                    <InformationValue>{user.username}</InformationValue>
+                  </InformationHolder>
+                  <InformationHolder>
+                    <InformationTitle>
+                      {intl.formatMessage(userFormMessages.userDevice)}
+                    </InformationTitle>
+                    <InformationValue>{user.device}</InformationValue>
+                  </InformationHolder>
```

Comment:
We would prefer to use the `Summary` component here instead of this. You can refer to the storybook to get an idea of how to use it. @modupeadeonojobi @sadmananik

Reply:
Alright